app/views/ChangeAvatarView/AvatarUrl.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import I18n from '../../i18n';
+import { FormTextInput } from '../../containers/TextInput';
+import { useDebounce, isImage, isValidURLRequest, isValidURL } from '../../lib/methods/helpers';
+
+const AvatarUrl = ({ submit }: { submit: (value: string) => void }) => {
+	const handleChangeText = useDebounce(async (value: string) => {
+		if (isImage(value) && isValidURL(value)) {
+			const result = await isValidURLRequest(value);
```

Comment:
Why do you need this?

Reply:
I'm testing if it's a valid URL, before changing the textURL and in sequence forcing the FastImage to render the URL